src/main/java/edu/ksu/canvas/attendance/submitter/CanvasAssignmentAssistant.java

```diff
@@ -59,6 +59,7 @@ public void createAssignmentInCanvas(Long courseId, AttendanceAssignment attenda
 
         LOG.info("Created canvas assignment: " + canvasAssignmentOptional.get().getId());
         saveCanvasAssignmentId(courseId, canvasAssignmentOptional.get());
+        return canvasAssignmentOptional.get();
```

Comment:
Also I don't think you introduced this, but .get() is often bad anyway.

https://reversecoding.net/java-8-optional-replace-get-examples/